src/elements/content-picker/Footer.js

```diff
@@ -102,6 +102,7 @@ const Footer = ({
                                 isDisabled={!selectedCount}
```

Comment:
feels kinda weird that `isDisabled` and `disabled` don't do the same thing, but I think it's too risky combining them, since we've been using `isDisabled` for a while, without setting `disabled` on the html element.
https://github.com/box/box-ui-elements/blob/fef7991a3e75ef8e437e1c2293ed1c92de907861/src/components/button/Button.tsx#L49-L58